ReactWindows/ReactNative.Shared/Modules/Core/ReactChoreographer.cs

```diff
@@ -223,24 +244,28 @@ private void OnRendering(object sender, object e)
 
         private void OnTick(object state)
         {
-            DispatcherHelpers.RunOnDispatcher(() =>
-            {
-                bool isSubscribed;
-                lock (_gate)
+            DispatcherHelpers.RunOnDispatcher(
+#if WINDOWS_UWP
```

Comment:
Its a little bit confusing to see a param in a call #ifed. I think two separate calls will be more readable

Reply:
I'll use an "else" for similar dispatcher (though it is always "main"). 2 calls may work, but the lambda expression is kind of long.

---
In reply to: [166084438](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r166084438) [](ancestors = 166084438)